src/main/java/com/google/api/codegen/discovery/Method.java

```diff
@@ -138,7 +138,7 @@ public Node parent() {
   /** @return the HTTP method. */
   public abstract String httpMethod();
 
-  /** @return the ID. */
+  /** @return the ID. This is unique. */
```

Comment:
Unique in what context?

Reply:
Updated to say within the context of the parent Document.